ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py

```diff
@@ -1292,3 +1292,40 @@ def test_torch_diag(
         input=values[0],
         diagonal=diagonal,
     )
+    
+ 
+@handle_frontend_test(
+    fn_tree="torch.bucketize",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),

```

Comment:
Why are we restricting the arrays generated to float only? I suggest using `valid` instead